src/InvestmentManager.sol

```diff
@@ -328,7 +328,10 @@ contract InvestmentManager is Auth {
             liquidityPool, _maxDeposit(liquidityPool, user) + currencyPayout, state.maxMint + trancheTokenPayout
         );
         state.maxMint = state.maxMint + trancheTokenPayout;
-        state.pendingDepositRequest = remainingInvestOrder;
+        state.pendingDepositRequest =
```

Comment:
wouldn't you simply use the currencyPayout for that? 
Why do we need the `fulfilledInvestOrder` variable?